### PR TITLE
Make a web page for the Fall 2020 meeting

### LINKED ISSUE #127: Create website for the PyHC Fall 2020 meeting
We need to put together a website for the Fall 2020 meeting ([see example here for Spring 2020 meeting](http://heliopython.org/meetings/april2020/)). Links to the various, relevant docs can be found in the Fall 2020 meeting Google Drive folder ([see here](https://drive.google.com/drive/u/0/folders/1T3CGRwXAst8jd7I6xFiKxyCgluGGpg0A)). Also, add meeting website link to the Meetings tab on the PyHC website (http://heliopython.org/meetings/).

### PR DESCRIPTION
**Pull request by sapols**
Closes #127. 

This PR adds a page to the PyHC site with information about the Fall 2020 meeting. I link to the page in the community announcement on the Meetings page, as well as in the blog post about it.

The new page was modeled after the Spring 2020 page. I list the dates and times of the seven sessions, link to all the relevant material in Google Drive, and place the Zoom/registration info at the bottom.

My one question is whether we think it's necessary to detail all the sessions here in this page. The Spring page included details about the tutorial session that took place after the meeting, but that was only one extra piece of info. I worry that if we add info about seven different things, the page will get cluttered and the Zoom/registration info at the bottom will become obscured. Is it enough that we link to this info in Google Drive?

### CODE REVIEW

File: _pages/meetings/fall2020.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Presentations and documents associated will be kept in the [**meeting folder**](https://drive.google.com/drive/u/0/folders/1T3CGRwXAst8jd7I6xFiKxyCgluGGpg0A) on Google Drive.
+
+ - [Meeting schedule](https://docs.google.com/spreadsheets/d/18lrxi8XsjDZ76MjWZPFOovu1JWrdjvoLZmyZfS_LZ3A/edit#gid=0) (subject to change)
+ - [Hackathon projects](https://docs.google.com/spreadsheets/d/1vPgf9nKnJaJ9kYZENxzD83UF84cwi4LQz1_vFLj9Sms/edit#gid=0)
+ - [Presentation, Tutorial, and Unconference Organizing](https://docs.google.com/spreadsheets/d/1dSLU4okGHOlPXGe969C2t9nQWlBr7NOP_F84Hn14WRw/edit#gid=0)
+
+### Session Details?
+
+Could include info about the seven individual sessions here...
```
Comment (sapols) [inline _pages/meetings/fall2020.md:30]: @jibarnum This is what I talked about in the PR description. What do you think?

Comment (jibarnum) [reply _pages/meetings/fall2020.md:30]: Yeah, the Spring 2020 meeting definitely was more pared down than what we're doing for the Fall 2020 meeting ==> way less information to be communicated on that meeting page than this one. I think it'd be fine to link to the info on Google Drive.

**Review by jibarnum — CHANGES_REQUESTED** [review]
LGTM, apart from adding in links to the Google Drive for session-specific info. 👍

File: _pages/meetings/fall2020.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+ - **Hackathons:** Nov 4th 7:00 AM–10:00 AM (MT)
+ - **Plenary #3:** Nov 9th 9:00 AM–10:00 AM (MT)
+ - **Unconferences:** Nov 11th 7:00 AM–10:00 AM (MT)
+ - **Plenary #4:** Nov 16th 9:00 AM–10:00 AM (MT)
+
+See [the full agenda](https://docs.google.com/spreadsheets/d/18lrxi8XsjDZ76MjWZPFOovu1JWrdjvoLZmyZfS_LZ3A/edit#gid=0) for exact dates/times of the various sessions. 
+
+### Presentations and Documents
+
+Presentations and documents associated will be kept in the [**meeting folder**](https://drive.google.com/drive/u/0/folders/1T3CGRwXAst8jd7I6xFiKxyCgluGGpg0A) on Google Drive.
```
Comment (sapols) [inline _pages/meetings/fall2020.md:22]: @jibarnum did I miss any links in this section? This line links to the Drive and the three below it link to the only materials I saw.

File: _pages/meetings/fall2020.md (hunk 1 above)
Comment (sapols) [reply _pages/meetings/fall2020.md:30]: Same page then. I’ll wack this Session Details section.

**Comment by sapols** [comment]
> LGTM, apart from adding in links to the Google Drive for session-specific info. 👍

I think I have done this already in the “Presentations and Documents” section. Maybe I should rename that header?

File: _pages/meetings/fall2020.md (hunk 1 above)
Comment (sapols) [reply _pages/meetings/fall2020.md]: Done.

**Review by jibarnum — APPROVED** [review]

File: _pages/meetings/fall2020.md (hunk 2 above)
Comment (jibarnum) [reply _pages/meetings/fall2020.md]: Nope you caught them all I think

**Pull request merged**